### DISCUSSION TITLE
Function keys functionality reversed

### DISCUSSION THREAD

**Discussion by arpeasal** [i3/i3#5428] [Q&A] [answered]
This is certainly not a major problem, but I’d like to understand what’s going on.

I bought a new Acer Aspire 5 to replace an older Acer Aspire 5.  I don’t know if the “aberration” described below is because the new laptop came with Endless OS (based on Linux) and the old one originally came with Windows.  On both I replaced the OS with Arch Linux and i3.

The “abberation” is that, for some reason the function keys seem to work in reverse.  That is, on the old computer, to raise or lower the volume, I press the Fn key at the same time as either the F10 or F11 key.  Pressing the F11 key WITHOUT the Fn key toggles full-screen mode.  On the new computer, this is reversed-I DON’T use the Fn key for volume control and I have to use it to toggle full-screen mode.

I am using exactly the same i3 config file on both.  (I just copied it from the old computer to the new one.)

When I first tried lowering the volume before I knew about the “aberration” and the computer went into full-screen mode, I started doing some research.  I ended up looking at the key mapping for both computers (using xmodmap –pke).  The key mapping for the volume control buttons (F9-mute, F10-volume down, F11-volume up) is the same on both computers.  These are keycodes 75, 76 and 77.  Keycodes 121, 122 and 123 are the codes for XF86Audio... And they are also the same on both computers.

So why is the way these keys work related to using or not using the Fn reversed?

**Comment by Airblader** [comment, marked as answer]
In Lenovos this is a BIOS setting, so I'd check there.

**Comment by arpeasal** [comment]
That was it! Thanks.
